src/Resolver/ObjktResolver.ts

```diff
@@ -47,6 +47,30 @@ export class ObjktResolver {
 		return ctx.genTokLoader.load(objkt.issuerId)
 	}
 
+	@FieldResolver((returns) => String, {
+		description: "IPFS uri pointing to the web page hosting the Generator code.",
+	})
+	artifactUri(@Root() objkt: Objkt, @Ctx() ctx: RequestContext) {
```

Comment:
This resolver should be removed as it can be derived from the Generative Token `generativeUri` and the objkt `generationHash`

`https://gateway.fxhash.xyz/ipfs/${cid(generativeUri)}/?fxhash=${generationHash}`